Level-0/00-Tools-of-the-Trade/Servers.md

```diff
@@ -419,16 +419,11 @@ Why use a database? If there is data that you want to hold on to and access late
 Things like passwords, account information, names, birthday, anything that you want to keep for a long time  all
 things stored in a database.
 
-A database organizes the data stored in it in a logical way. This allows for the best efficiency when storing or retrieving
-the data from it. The database is organized into `tables`. Each `table` has columns and rows. You could think of it as a
-sheet of graph paper. The rows (the lines going from left to right) are called `records`. The individual cells that hold
-data are called `fields`.
+A database organizes the data stored in it in a logical way.  This allows for the best efficiency when storing or retrieving the data from it.  The database is organized into 'tables'.  A 'table' is a rectangular configuration of 'cells'.  You could think of it as a sheet of graph paper.  A column of cells of a particular data type (text, number, date, etc.) is referred to as a 'field', typically given some meaningful label, such as: 'FirstName', and will be referred to as the 'FirstName field'.  Other such fields may be 'LastName', 'DateOfBirth', etc.  Each cell is then filled with a 'value' that corresponds to that 'field'.  Each row is called a 'record'.  Information is typically entered one record (row) at a time, field by field.  The value in a cell is simply that, the 'value'.  A database can have more that one 'table'.
```

Comment:
Thanks, @daveissomewhere. There are a few things to address here. 
* lines should be limited to 125 characters, use newlines to break sentences/paragraphs onto new liens as needed.
* paragraphs and sections should be separated by a single empty line (you already did this)
* the images should e stored in the repository for offline viewing (you replaced a relative file path with a URI which is only accessible via the Internet).
* You replaced the backtick with a single quote. I think the intention of the backtick was to visually highlight the word. Maybe that isn't the right thing to do, but if we change it here we should change it everywhere that it used for that purpose.

There is a tool, Markdownlint, that should be run before each commit is allowed to be created. Markdownlint checks your Markdown files for issues based on a configuration file created to defined what we want our Markdown to look like.

A "lint", or "linter", is a code analysis tool used to detect some errors, bugs, stylistic errors, and suspicious constructs. The term comes from an old 70's Bell Labs tool for Unix called `lint` that examined C source code.

> The term "lint" was derived from lint, the name for the tiny bits of fiber and fluff shed by clothing, as the command should act like the lint trap in a clothes dryer, detecting small errors to great effect.
(from [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software)))

When used as a verb is, it means to run your linter or handle the issues found. 

 Since this is done on your local machine by running some JavaScript code, it doesn't happen when run in GitHub. You should pull down the repository to your local machine and make your changes there.